### ISSUE TITLE
EMC Error log research

### ISSUE THREAD

**Issue by andy-man** [andy-man/ps5-wee-tools#1]
Post your codes here. Describe in details about problems that you encounter during this repair.

**Comment by RustyRaindeer** [comment]
Gor this one with a DDR6 RAM module (slot num#5) unsoldered:

OK 00000000 **80801110** 0092611B FFFF0005 00000100 2013 0016 1A36 0000:F8

Just took note of it while troubleshooting a BLOD console...

**Comment by naif00700** [comment]
Yes, the error was due to DDR6 (2). All errors must be cleared and restarted only once. When trying to restart more than once, for the third time, a blue light flashes continuously and the following symbols appear:
<img width="734" height="552" alt="Image" src="https://github.com/user-attachments/assets/84c9d659-93ab-4ebd-8cc1-a931839dc5ba" />
<img width="730" height="296" alt="Image" src="https://github.com/user-attachments/assets/a25cf3d5-1e76-41e6-80f2-39972dcb66d2" />

**Comment by naif00700** [comment]
After replacing the DDR6 (2) it works perfectly for more than an hour without stopping

**Comment by simkard69** [comment]
```
 ## Code     Rtc      PowState UpCause  SeqN DvPm TSoC TEnv
----------------------------------------------------------------------

 01 80931113 000A840B FFFF0001 10000000 2179 FFFF FFFF 0024

 Date             : 2012-01-08 23:26:03
 OS state         : INIT
 System state     : STANDBY
 Last Sequence    : 2179[8569] - BT WAKE Enabled
 Power state      : HDMI_CEC
 Temperatures     : -В° (SoC) / -В° (Env)

 Unknown code - please report

----------------------------------------------------------------------

 02 80931113 000A840B FFFF0001 10000000 2179 FFFF FFFF 0024

 Date             : 2012-01-08 23:26:03
 OS state         : INIT
 System state     : STANDBY
 Last Sequence    : 2179[8569] - BT WAKE Enabled
 Power state      : HDMI_CEC
 Temperatures     : -В° (SoC) / -В° (Env)

 Unknown code - please report
```

I have a console with these error showing up, anyway, it doesn't avoid the console to work except that it fails reset ps5 from safe mode.
Even if I manage to get things installed and booted, no luck with registering the disc drive and/or playing any game, except astro's playground which ends up crashing at the location systematically.
At that stage, I didn't find the problem and I'm stuck at that stage.

**Comment by naif00700** [comment]
I think these errors are software errors, not hardware errors. You should
try installing the entire firmware and trying again. If the same errors
recur, you should replace DDR 1, 2, 3, 4, 5, 6, 7, and 8 all at once and
see the results.

في الاثنين، ٢٨ يوليو ٢٠٢٥، ٣:١١ م simkard69 ***@***.***> كتب:
